roles/ansible-test-splitter/files/split_targets.py

```diff
@@ -59,7 +59,7 @@ def get_targets_to_run(targets, targets_from_cli):
     for target_name, lines in targets.items():
         if to_skip_because_of_targets_parameters(target_name, lines, targets_from_cli):
             continue
-        if to_skip_because_disabled(lines):
+        if to_skip_because_disabled(lines) and not targets_from_cli:
```

Comment:
You also include disabled target. Is this something expected? The PR title just mentions the unstable targets.

Reply:
Also, if `targets_from_cli` is empty, we should ignore the filter. I think we should instead have something like:

`if to_skip_because_unstable(lines) and target_name not in targets_from_cli:`